### PR TITLE
Add license check for Java files

### LINKED ISSUE #56: Test that all source files have licenses

### PR DESCRIPTION
**Pull request by pongad**
Fixes #56.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -291,4 +293,33 @@ project(':vgen') {
       args = clargs.split(',').toList()
     }
   }
+
+  task verifyLicense << {
+    def licenseText = '''/* Copyright 2016 Google Inc
```
Comment (garrettjonesgoogle) [inline build.gradle]: It would be nice if we could derive the check from a file that was outside of this build file.

Comment (pongad) [inline build.gradle]: I uploaded a change to move the license into its own file. It feels a little funny to have it in the top-level, but I'm not sure where else to put it. Let me know what you think.

File: java-license.txt [hunk 2]
```diff
@@ -0,0 +1,14 @@
+/* Copyright 2016 Google Inc
```
Comment (garrettjonesgoogle) [inline java-license.txt]: This file isn't the license for java; maybe call the filename "license-header-javadoc.txt"

Comment (pongad) [inline java-license.txt]: That's what I meant, but I now see how the name is confusing. Will fix this.

**Comment by pongad** [comment]
@garrettjonesgoogle PTAL

**Comment by garrettjonesgoogle** [comment]
+1; have one more person review it.

**Comment by saicheems** [comment]
lgtm

**Pull request merged**